pkg/flag/clean_flags.go

```diff
@@ -0,0 +1,83 @@
+package flag
+
+var (
+	CleanAll = Flag[bool]{
+		Name:       "all",
+		ConfigName: "clean.all",
+		Usage:      "remove all caches",
+	}
```

Comment:
What do you think about short name? 
```suggestion
	CleanAll = Flag[bool]{
		Name:       "all",
		ConfigName: "clean.all",
		Usage:      "remove all caches",
	}
```

Reply:
What is your suggestion? `Suggested change` looks identical to me. Do you mean `-a` or something similar?

Reply:
hm... looks like i saved suggestion incorrectly.

 > . Do you mean -a or something similar?

Yes. `-a` is usual shorthand for `--all` flag

Reply:
Added https://github.com/aquasecurity/trivy/pull/6993/commits/ba53b672d65d191557f3a6a288d839fd3af550a7